plugins/11_TheWarWithin/localization/enUS.lua

```diff
@@ -249,6 +249,8 @@ L['nightfarm_growthling_note'] = 'Use 3x {item:206350} at the {object:Lesser Key
 
 L['thunder_lynx_note'] = '1. Use 3x {item:206350} at the {object:Lesser Keyflame} in {location:Torchlight Mine} to summon {npc:212419}.\n\n2. Speak with {npc:212419} and follow all extra dialog prompts to reveal {quest:82007}.\n\n3. Locate each baby lynx: {npc:222373} ({dot:Blue}), {npc:222372} ({dot:Green}), {npc:222375} ({dot:Orange}), and {npc:222374} ({dot:Red}).\n\n{note:Be sure to start {quest:76169} at the same to to receive {item:219198} or use a similar item such as {item:219148}. You\'ll need a light to reveal {npc:222373} within {location:Coldshadow Cave}.}\n\n4. Rescue each lynx kitten and return to {npc:212419}.'
 
+L['nightfall_elites_note'] = 'A rare elite may spawn directly or after clicking the {object:Suspicious Document}.'
```

Comment:
The current description is correct: there are 2 rares per assault point, one of which requires clicking a document, while the other spawns directly without needing to click. This original description is accurate.